### PR TITLE
c03 finished translation by 9TDWorld

### PR DESCRIPTION
**Pull request by 9TDWorld**
Removed all zz files for MT. Updated all untranslated lines. 
Early commit lead to an odd moving of a file that messed up repo. Deleted and reuploaded original file.

### CODE REVIEW

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_answer/translation.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「ま、まさかっ……エッチしたいなんて……」=
-//「も、もちろん……いいですよ……？　その、い、今こんなことしてたですから……私だって、準備万端……なのです」=
+「ま、まさかっ……エッチしたいなんて……」=“N-no way ...... you want to have sex ......”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_answer/translation.txt:4]:
```suggestion
「ま、まさかっ……エッチしたいなんて……」=“N-no way... you want to have sex...”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_answer/translation.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
+「も、もちろん……いいですよ……？　その、い、今こんなことしてたですから……私だって、準備万端……なのです」=“O-of course ...... it's fine ......? Well, s-since I was just doing this ...... I'm ready ...... as well”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_answer/translation.txt:5]:
```suggestion
「も、もちろん……いいですよ……？　その、い、今こんなことしてたですから……私だって、準備万端……なのです」=“O-of course... it's fine...? Well, s-since I was just doing this... I'm ready... as well” 
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/31/31/translation.txt [hunk 3]
```diff
@@ -1,7 +1,7 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.2 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「風が気持ちいいのですー！　嫌なことがあった時にここに来たら、スッキリ出来そうですね」=
+「風が気持ちいいのですー！　嫌なことがあった時にここに来たら、スッキリ出来そうですね」=“The wind feels so goodー！ You can come here when you've had a bad day and feel refreshed.”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/31/31/translation.txt:4]:
```suggestion
「風が気持ちいいのですー！　嫌なことがあった時にここに来たら、スッキリ出来そうですね」=“The wind feels so good！ You can come here when you've had a bad day and feel refreshed.”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/heroine_action/translation.txt [hunk 4]
```diff
@@ -2,5 +2,5 @@
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
 「えっと……混ざりたいの？　……どうする？」=“Erhm... You wanna join us? What do you wanna do?”
-//「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=
-//「仕方ないです、ここは既成事実を作るしかないです！」=
+「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=“Mmm ...... after being seen like this, we're already in this together!”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/heroine_action/translation.txt:5]:
```suggestion
「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=“Mmm... after being seen like this, we're already in this together!”
```

Also, maybe reword a bit:

```suggestion
「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=“Mmm... now that we've been seen like this, we're in this together!”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/heroine_end/translation.txt [hunk 5]
```diff
@@ -1,4 +1,5 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「二人きりでするのとは全然違う感じで……これはこれでまたやりたいのです♪」=
+「二人きりでするのとは全然違う感じで……これはこれでまたやりたいのです♪」=“It feels completely different from doing it with just the two of us ...... I want to do this again♪”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/heroine_end/translation.txt:4]:
```suggestion
「二人きりでするのとは全然違う感じで……これはこれでまたやりたいのです♪」=“It feels completely different from doing it with just the two of us... I want to do this again♪”
```

Also, sounds a bit awkward. maybe: 

```suggestion
「二人きりでするのとは全然違う感じで……これはこれでまたやりたいのです♪」=“It feels completely different from just the two of us doing it... I want to do this again♪”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_answer/translation.txt [hunk 6]
```diff
@@ -1,5 +1,5 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=
-//「仕方ないです、ここは既成事実を作るしかないです！」=
+「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=“Mmm ...... after being seen like this, we're already in this together!”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_answer/translation.txt:4]: same as above

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_end/translation.txt [hunk 7]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=
+「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=“I can't believe how fun it is to have sex with everyone ...... I've fallen in love with it!”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_end/translation.txt:4]:
```suggestion
「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=“I can't believe how fun it is to have sex with everyone... I've fallen in love with it!”
```

maybe make the second bit sound less passive?

```suggestion
「みんなでのエッチがこんなに楽しいなんて……私、もうめろめろになったですよ！」=“I can't believe how fun it is to have sex with everyone... I love it!”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_reaction/translation.txt [hunk 8]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「はわー！　ま、まさか私達のれずれずえっちを見られてたなんて……」=
+「はわー！　ま、まさか私達のれずれずえっちを見られてたなんて……」=“Hawaー! I can't believe you were watching ......”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_reaction/translation.txt:4]:
```suggestion
「はわー！　ま、まさか私達のれずれずえっちを見られてたなんて……」=“Hawaー! I can't believe you were watching...”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation/translation.txt [hunk 9]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「ひあぁっ、そんなところで何してるですかーっ！？」=
+「ひあぁっ、そんなところで何してるですかーっ！？」=“Hiaah! What are you doing there！?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation/translation.txt:4]: replace wide punctuation characters:
```suggestion
「ひあぁっ、そんなところで何してるですかーっ！？」=“Hiaah! What are you doing there!?”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_found/translation.txt [hunk 10]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「ふふーっ、見てるだけじゃ物足りないんじゃないですか？　あなた……すっごくえっちなお顔してるですよ？」=
+「ふふーっ、見てるだけじゃ物足りないんじゃないですか？　あなた……すっごくえっちなお顔してるですよ？」=“Fufu, I guess it wasn't enough to just watch? You ...... have a very naughty face don't you?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_found/translation.txt:4]:
```suggestion
「ふふーっ、見てるだけじゃ物足りないんじゃないですか？　あなた……すっごくえっちなお顔してるですよ？」=“Fufu, I guess it wasn't enough to just watch? You... have a very naughty face don't you?”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_found_answer/translation.txt [hunk 11]
```diff
@@ -1,5 +1,5 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「わかってるです、そんなに大きくしちゃってるですから……」=
-//「それじゃ、早速するです！　ほら、するったらするですよ！」=
+「わかってるです、そんなに大きくしちゃってるですから……」=“I know, it's gotten so big. ......”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/masturbation_found_answer/translation.txt:4]:
```suggestion
「わかってるです、そんなに大きくしちゃってるですから……」=“I know, it's gotten so big...”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/penetrator_action/translation.txt [hunk 12]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「あのっ、私もご一緒させてもらえないですか？　その……二人を見てたら、ぬるぬるになっちゃったのです……」=
+「あのっ、私もご一緒させてもらえないですか？　その……二人を見てたら、ぬるぬるになっちゃったのです……」=“Hey, can I join you? Um .......I got so wet watching you two.......”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/penetrator_action/translation.txt:4]:
```suggestion
「あのっ、私もご一緒させてもらえないですか？　その……二人を見てたら、ぬるぬるになっちゃったのです……」=“Hey, can I join you? Um... I got so wet watching you two...”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/penetrator_action_end/translation.txt [hunk 13]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「あ、ありがとです！　それじゃ早速……えっと、どうやってするですか……？」=
+「あ、ありがとです！　それじゃ早速……えっと、どうやってするですか……？」=“T-thanks! Then let's get started ...... um, how do we do this?”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/penetrator_action_end/translation.txt:4]:
```suggestion
「あ、ありがとです！　それじゃ早速……えっと、どうやってするですか……？」=“T-thanks! Then let's get started... um, how do we do this?”
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/penetrator_end/translation.txt [hunk 14]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「えへへ、入れてもらえて嬉しかったです。おかげでムラムラもおさまったですよー♪」=
+「えへへ、入れてもらえて嬉しかったです。おかげでムラムラもおさまったですよー♪」=“Hehe, I'm so you let me join. Thanks to that my horniness has calmed.ー♪"
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/penetrator_end/translation.txt:4]:
```suggestion
「えへへ、入れてもらえて嬉しかったです。おかげでムラムラもおさまったですよー♪」=“Hehe, I'm so glad you let me join in. Thanks to that my horniness subsided!♪"
```

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/sub_end/translation.txt [hunk 15]
```diff
@@ -1,4 +1,4 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「はぁ、わかってたです……あんなの、絶対いいってわかってたですよぉ……」=
+「はぁ、わかってたです……あんなの、絶対いいってわかってたですよぉ……」=“Haaa, I knew it ...... I knew it would be fantastic ......”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/sub_end/translation.txt:4]:
```suggestion
「はぁ、わかってたです……あんなの、絶対いいってわかってたですよぉ……」=“Haaa, I knew it... I knew it would be fantastic...”
```

**Review by GeBo1 — CHANGES_REQUESTED** [review]
Thanks!  Needs a few fixes, mostly standardizing on English style punctuation, but I suggested a few rewordings as well.

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_end/translation.txt (hunk 7 above)
Comment (9TDWorld) [reply Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_end/translation.txt:4]: I went with fallen in love to keep consistency with some of the other uses of meromero both from KK's kouhai character and one of the other usages in a different conversation. I like that it has a slightly different nuance than just saying she loves it.

File: Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_answer/translation.txt [hunk 16]
```diff
@@ -1,5 +1,5 @@
 //
 // Dumped for Koikatsu Sunshine v1.0.1 (KoikatsuSunshine) by KKS_TextDump v1.4.5.0
 //
-//「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=
-//「仕方ないです、ここは既成事実を作るしかないです！」=
+「むぅぅ……あんなえっちなとこ見られてたら、もう一蓮托生です！」=“Mmm... after being seen like this, we're already in this together!”
```
Comment (GeBo1) [inline Translation/en/RedirectedResources/assets/abdata/adv/scenario/c03/penetration/main_answer/translation.txt:4]: Doesn't match the one from [penetration/heroine_action/translation.txt](https://github.com/IllusionMods/KKS-Translation/pull/146/files#diff-9ed6e5c24dc6b6d8107cdcda32e46fbbccf6ede413e66dff5dc6a0408ea4c547), update this one to match

**Review by GeBo1 — APPROVED** [review]

**Review by ManlyMarco — APPROVED** [review]
